usaspending_api/common/helpers/api_helper.py

```diff
@@ -87,12 +88,18 @@ def raise_if_sort_key_not_valid(sort_key, field_list, is_subaward=False):
         msg_prefix = "Sub-"
         field_external_name_list = list(contract_subaward_mapping.keys()) + list(grant_subaward_mapping.keys())
     else:
-        field_external_name_list = (
-            list(contracts_mapping.keys())
-            + list(loan_mapping.keys())
-            + list(non_loan_assist_mapping.keys())
-            + list(idv_mapping.keys())
-        )
+        if set(award_type_codes) <= set(contract_type_mapping):
```

Comment:
There's a chance `field_external_name_list` will be uninitialized since these set of conditions do not have a base/default case.

Reply:
Done